### ISSUE TITLE
Slope attribute stripped on upload

### ISSUE THREAD

**Issue by bobsmith1981** [darylldoyle/safe-svg#1]
I've built a simple filter for adding a semi transparent drop shadow to shapes in an SVG, works great locally but when I uploaded it to my WordPress site with your plugin installed I noticed it removed the slope attribute from feFuncA element.

Uploaded Code:
```
<filter id="f2" x="0" y="0" width="200%" height="200%">
      <feOffset result="offOut" in="SourceAlpha" dx="5" dy="5" />
      <feGaussianBlur result="blurOut" in="offOut" stdDeviation="5" />
      <feComponentTransfer in="blurOut" result="alphaOut">
        <feFuncA type="linear" slope="0.2"/>
      </feComponentTransfer>
      <feBlend in="SourceGraphic" in2="alphaOut" mode="normal" result="shadow" />
</filter>
```

Code from SVG on Site:
```
<filter id="f2" x="0" y="0" width="200%" height="200%">
	<feOffset result="offOut" in="SourceAlpha" dx="5" dy="5"></feOffset>
	<feGaussianBlur result="blurOut" in="offOut" stdDeviation="5"></feGaussianBlur>
	<feComponentTransfer in="blurOut" result="alphaOut">
		<feFuncA type="linear"></feFuncA>
	</feComponentTransfer>
	<feBlend in="SourceGraphic" in2="alphaOut" mode="normal" result="shadow"></feBlend>
</filter>
```

Here is a gist for the full SVG image:
https://gist.github.com/bobsmith1981/b54af9259370fba821bdc5027ffedef2#file-velpeo-svg-logo

Here is a version missing the slope:
http://cdn.velpeo.com/wp-content/uploads/2018/09/30152059/large-square-icon-minimal-01.svg

**Comment by darylldoyle** [comment]
This has been fixed in 1.7.1. Will release on the WP.org directory shortly!

Also closes #2

**Comment by bradmkjr** [comment]
Thank you
